addons/website/models/ir_http.py

```diff
@@ -442,6 +442,7 @@ def get_frontend_session_info(self):
         geoip_phone_code = request.env['res.country']._phone_code_for(geoip_country_code) if geoip_country_code else None
         session_info.update({
             'is_website_user': request.env.user.id == request.website.user_id.id,
+            'lang_url_code': request.lang.url_code,
```

Comment:
Because the url_code is often used, website uses an advanced mechanism to cache the value. To benefit from the cache you need to use `_get_cached`.
```suggestion
            'lang_url_code': request.lang._get_cached('url_code'),
```